service/src/test/java/uk/gov/hmcts/reform/fpl/service/search/SearchServiceTest.java

```diff
@@ -64,4 +60,42 @@ void shouldSearchCasesByDateProperty() {
         JSONAssert.assertEquals(queryCaptor.getValue(), expectedQuery, NON_EXTENSIBLE);
     }
 
+    @Test
+    void shouldSearchCasesWhenGivenESQuery() {
+        ESQuery query = BooleanQuery.builder()
+            .mustNot(MustNot.builder().clauses(List.of(MatchQuery.of("a", "b"))).build())
+            .build();
+
+        when(coreCaseDataService.searchCases(any(), any())).thenReturn(EXPECTED_CASES);
+
+        List<CaseDetails> casesFound = searchService.search(query);
+
+        assertThat(casesFound).isEqualTo(EXPECTED_CASES);
+
+        verify(coreCaseDataService).searchCases(eq("CARE_SUPERVISION_EPO"), queryCaptor.capture());
```

Comment:
you can use:
verify(coreCaseDataService).searchCases(eq("CARE_SUPERVISION_EPO"), eqJson(expectedQuery));